api/features/serializers.py

```diff
@@ -36,8 +41,29 @@ def remove_users(self, feature: Feature):
         feature.owners.remove(*user_ids)
 
 
+class FeatureGroupOwnerInputSerializer(serializers.Serializer):
+    group_ids = serializers.ListField(child=serializers.IntegerField())
+
+    def validate(self, data):
+        if not UserPermissionGroup.objects.filter(
+            id__in=data["group_ids"]
+        ).count() == len(data["group_ids"]):
+            raise serializers.ValidationError("Some groups not found")
+
+        return data
```

Comment:
We may want to consider validating that the groups belong at least to the organisation for the feature we are adding them to.

Reply:
Ok this has been implemented.